openmeetings-db/src/main/java/org/apache/openmeetings/db/entity/file/BaseFileItem.java

```diff
@@ -347,9 +347,18 @@ public final File getOriginal() {
 		if (f != null) {
```

Comment:
Sorry for the double post :(

The simpler solution would be: to replace above line
`File f = getFile(null);`
with
`File f = getFile(Type.PRESENTATION == type ? EXTENSION_PDF : null);`

This should work without any other modifications ...

Reply:
Done. Manual and automated tests seem to pass.

Reply:
U are happy with it?